tests/crypto/README.rst

```diff
@@ -0,0 +1,306 @@
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HMAC SHA512 |                                                                            | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HKDF               | HKDF SHA256 | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HKDF SHA512 |                                                                            | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| EC-JPAKE           | secp256r1   | `J-PAKE: Password-Authenticated Key Exchange by Juggling`_                 | Custom                                                                     |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF9160DK|
+  * |nRF52840DK|
+  * |nRF52DK|
+  * |nRF51DK|
+
+.. note::
+   Nordic devices with smaller flash capacity, such as nRF51, nRF52810, or nRF52811 might not fit the full test suite.
+   A recommended approach in such case is to run subsets of the tests one by one.
+
+
+Building and running
+********************
+
+.. |test path| replace:: :file:`tests/crypto/`
+
+.. include:: /includes/build_and_run_test.txt
+
+There are multiple ways to build the tests.
+See :ref:`nrf_security_readme` for additional information about nrf_security configuration.
+You can use the following configuration files to build it in a specific setup:
+
+* :file:`overlay-backend-sw.conf` - software only, except a hardware module to generate entropy for RNG
+* :file:`overlay-backend-hw.conf` - hardware-accelerated using the Arm CryptoCell accelerator (for cryptography and entropy for RNG)
+* :file:`overlay-backend-hw-sw.conf` - a combination of hardware acceleration using Arm CryptoCell and a software implementation that adds key sizes and algorithms not supported in CryptoCell (the setup uses hardware as much as possible)
+
```

Comment:
`overlay-backend-oberon.conf` got merged, should have an entry as well

Reply:
> RTT for output is mentioned several times in the doc.
> Note that in the current master, output only shows up on UART.
> I made a little PR for this: #2362
> 
> When it's in, both UART and RTT will be on.

Oh. that is a little ouchie. Maybe we can write something along the lines of 

"will be outputed on either UART or RTT depending on configuration"

Reply:
@frkv  what would the description be for the nrf_oberon configuration?

Reply:
done, got a description from Torstein